web-app/src/main/webapp/css/style.css

```diff
@@ -1,9 +1,26 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+#map {
+  /* placeholder until we add a map */
```

Comment:
nit: I would mark TODOs clearly so that we don't forget about them. You can do `TODO(AshleyLTW)` or `TODO(jiminbyun)`